docs/tutorial/tutorial.md

```diff
@@ -216,7 +216,7 @@ class Square extends React.Component {
 }
 ```
 
-In [JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), you need to explicitly call `super();` when defining the constructor of a subclass.
+In [JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), you need to explicitly call `super();` when defining the constructor of a subclass. When creating a class component in React, you should always call the base constructor with `props`.
```

Comment:
Ah, that's true. It may not be necessary, but excluding props also might still be outside of the intended contract for the `React.Component` constructor. [Other docs do state that `props` should always be passed to super](https://facebook.github.io/react/docs/state-and-lifecycle.html#adding-local-state-to-a-class).

So there's inconsistency here. It might not matter, but I'm curious for @gaearon's take here. Is calling `super()` without passing props problematic for present or future versions of React? 

If a hard fast rule of "always all React.Component.constructor with `props`" isn't true, we should take a pass to clear up the ambiguity.